test-produsent/src/Komponenter/NyBeskjed.tsx

```diff
@@ -78,6 +88,42 @@ export const NyBeskjed: React.FunctionComponent = () => {
 
 
     const handleSend = () => {
+        const eksterneVarsler = eksternVarsel === "Ingen" ? [] : [{
```

Comment:
synes det er litt vanskelig å se disse ternary operatorene. I tillegg så blir det som nevnt en liste med ett objekt med alle keys satt, men to av verdiene null. I kontrakten brukes direktivet [@ExactlyOneFieldGiven](https://github.com/navikt/arbeidsgiver-notifikasjon-produsent-api/blob/6c653a25376e73bd7f9c5c82323b9f20caca9a0f/app/src/main/resources/produsent.graphql#L8) som tilsier at du må og kun kan angi en av feltene. Det vil fungere siden [validatoren i backend er implementert](https://github.com/navikt/arbeidsgiver-notifikasjon-produsent-api/blob/6c653a25376e73bd7f9c5c82323b9f20caca9a0f/app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/infrastruktur/graphql/GraphQLValidation.kt#L191-L209) til å filtrere ut keys med null verdi, men kan være litt vanskelig å vite eller være overraskende oppførsel. 

Kanskje det hadde vært lettere å lese med en if/else eller switch case som assigner verdien?